lib/dialects/mssql/query.js

```diff
@@ -51,7 +51,7 @@ class Query extends AbstractQuery {
           } else {
             resolve(this.formatResults());
           }
-        }, this.options.transaction.name, this.options.isolationLevel);
+        }, this.options.transaction.name, Utils.mapIsolationLevelStringToTedious(this.options.isolationLevel));
```

Comment:
@sushantdhiman `this.lib` will not be available here. `this.lib` is from `ConnectionManager`.